### PR TITLE
add seth---to-int256 macro

### PR DESCRIPTION
**Pull request by MrChico**
addresses https://github.com/dapphub/dapptools/issues/170

### CODE REVIEW

**Comment by MrChico** [comment]
Does anyone want to review before I merge @asymmetric @rainbreak ?

File: src/seth/libexec/seth/seth---to-int256 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+### seth---to-int256 -- convert an integer into int256 value
+### Usage: seth --to-int256 <value>
+set -e
+export BC_LINE_LENGTH=5000
+seth --to-uint256 $(bc <<< "(2^256 * ($1 < 0)) + $1")
```
Comment (asymmetric) [inline src/seth/libexec/seth/seth---to-int256]: [`shellcheck`](http://shellcheck.net/) gives a warning about missing quotes around the process substitution - could you do that?

I can recommend installing a `shellcheck` plugin locally btw, and I plan to add it to the CI pipeline.

File: src/seth/libexec/seth/seth---to-int256 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+### seth---to-int256 -- convert an integer into int256 value
+### Usage: seth --to-int256 <value>
+set -e
+export BC_LINE_LENGTH=5000
```
Comment (asymmetric) [inline src/seth/libexec/seth/seth---to-int256]: Is the actual max number of characters not `2 * bytes(word_size) = 64`?

File: src/seth/libexec/seth/seth---to-int256 (hunk 1 above)
Comment (livnev) [reply src/seth/libexec/seth/seth---to-int256]: 2nded, @MrChico I recommend installing `shellcheck` and `(add-hook 'sh-mode-hook 'flycheck-mode)`.

Comment (asymmetric) [reply src/seth/libexec/seth/seth---to-int256]: It's now run in Travis https://github.com/dapphub/dapptools/pull/175

Comment (MrChico) [reply src/seth/libexec/seth/seth---to-int256]: Thanks, should be fixed now

File: src/seth/libexec/seth/seth---to-int256 (hunk 2 above)
Comment (MrChico) [reply src/seth/libexec/seth/seth---to-int256]: This is just to force `bc` to not break lines. The actual length is however many digits 2^256 (in decimal) has. But 5000 suffices

Comment (rainbreak) [reply src/seth/libexec/seth/seth---to-int256]:
```
$ BC_LINE_LENGTH=5000 bc <<< 2^256 | wc -c
79
```

**Pull request merged**